src/test/java/SYSC6/Project/UserControllerTest.java

```diff
@@ -104,8 +104,8 @@ void list() throws Exception {
         JSONObject obj = new JSONObject();
         String jsonStringTexpected = "{\"data\":[],\"recordsFiltered\":5,\"recordsTotal\":5,\"draw\":0}";
         this.mockMvc.perform(post("/rest/api/usertable")
-                .contentType(MediaType.APPLICATION_JSON)
-                .content(String.valueOf(obj)))
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(String.valueOf(obj)))
                 .andExpect(status().is2xxSuccessful())
```

Comment:
unnecessary whitespace. please change this to avoid unnecessary merge conflicts for others